_config/model.yml

```diff
@@ -5,3 +5,5 @@ Name: markdownfieldtype
 SilverStripe\Core\Injector\Injector:
   MarkdownText:
     class: SilverStripers\markdown\db\MarkdownText
+  HTMLText:
+    class: SilverStripers\markdown\db\MarkdownText
```

Comment:
Enforcing it would be something of a choice for the developer instead of adding it in the module. I would suggest documenting how to make HTMLText go to MarkdownText, instead of making it the default behaviour.

Other than that, this looks pretty good to me!

Reply:
Yeah that makes sense. I've taken it out. and updated the readme with that info. Also did some additional changes on how it replaces the base Content field on SiteTree